addons/cssresources/src/CssResourcePanel.js

```diff
@@ -0,0 +1,155 @@
+import { document, DOMParser } from 'global';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { monoFonts } from '@storybook/components';
+import styled from '@emotion/styled';
+import 'pretty-checkbox/dist/pretty-checkbox.css';
+
+import css from 'react-syntax-highlighter/languages/prism/css';
```

Comment:
There some performance improvement @Hypnosphi did in storysource addon for the `react-syntax-highlighter`. Take a look there